app/views/sal3_batch_requests_batches/new.html.erb

```diff
@@ -127,9 +127,10 @@
     </div>
   </div>
   <div class='form-group row'>
-    <%= f.label :bc_file_obj, 'Upload barcode file (one barcode per line)', class: 'col-sm-2 form-control-label' %>
+    <%= f.label :bc_file_obj, 'Upload barcode file', class: 'col-sm-2 form-control-label' %>
     <div class='col-sm-10'>
       <%= f.file_field 'bc_file_obj', class: 'form-control-file' %>
+      Plain text file. Use Notepad or TextWrangler. One barcode per line. No punctuation.
```

Comment:
Sarah asks if this can be in italics.  Will be more obvious as help text that way.